adapter/list/list_test.go

```diff
@@ -387,7 +387,7 @@ func TestBadUrl(t *testing.T) {
 	info := GetBuilderInfo()
 	builder := info.CreateHandlerBuilder()
 	cfg := config.Params{
-		ProviderUrl:     "http://abadurl.com",
```

Comment:
It was failing tests because that url resolved to something.
The replacement is more guaranteed to be bad.